types.go

```diff
@@ -261,7 +261,12 @@ type VSPTicketPurchaseInfo struct {
 
 /** end ticket-related types */
 
-/** begin politea proposal types */
+/** begin politeia types */
```

Comment:
There's a lot of `int` usage in this file that doesn't seem necessary. Ints get converted to Long on gomobile and it's not really convenient to use so please use `int32` where possible.